pim-tool-fe/src/app/component/project-detail/project-detail.component.ts

```diff
@@ -0,0 +1,362 @@
+  languageList = [
+    { code: 'en', label: 'English' },
+    { code: 'fr', label: 'French' },
+  ];
+  groups: Group[] | undefined;
+  updateProject!: Project;
+  updateProjectMembers!: ProjectMembers;
+  actionTitle: any = 'projectDetail.create.title';
+  btnSubmitContent: any = 'projectDetail.create.btnCreate';
+  editMode: boolean = false;
+  numberErr: string = '';
+  ennDateErr: string = '';
+  globalErr: string = 'projectDetail.globalError';
+  membersError: string = '';
+  projectSent!: ProjectMembers;
+  isFailed: boolean = false;
+  empList: Employee[] = [];
+  selectedItem: Employee[] = [];
+  emptyMessage: "No employees found" | undefined;
+  formGroup: FormGroup | undefined;
+  selectedEmployee: number[] = [];
+  hasEmp: boolean = false;
+  startDate: any;
+  endDate: any;
+
+  constructor(
+    private projectService: ProjectService,
+    private groupService: GroupService,
+    private employeeService: EmployeeService,
+    private router: Router,
+    private route: ActivatedRoute,
+    public sharedService: SharedService,
+    private translate: TranslateService,
+    private primengConfig: PrimeNGConfig
+  ) { }
+
+  ngOnInit(): void {
+    this.primengConfig.ripple = true;
+    this.getGroups();
+    this.getEmployees();
+    this.globalErr = 'projectDetail.globalError';
+    const projectNumber: any =
+      this.route.snapshot.paramMap.get('projectNumber');
+    if (projectNumber) {
+      this.editMode = !this.editMode;
+      this.actionTitle = 'projectDetail.update.title';
+      this.btnSubmitContent = 'projectDetail.update.btnUpdate';
+      this.getProjectByNumber(projectNumber);
+    }
+    console.log('editMode: ', this.editMode);
+  }
+
+  getEmployees() {
+    this.employeeService.getEmployees().subscribe(
+      (response) => {
+        this.hasEmp = true;
+        this.empList = response
+        console.log("list employees: ", this.empList);
+      },
+      (error: HttpErrorResponse) => {
+        this.empList = []
+        this.hasEmp = false;
+        console.log("error get employees: ", error);
+        this.navigateToErrorPage();
+      }
+    )
+  }
+
+  search($event: any) {
+    this.employeeService.searchEmployees($event.query).subscribe(
+      (response) => {
+        this.empList = response,
+        this.membersError = ''
+      },
+      (error: HttpErrorResponse) => {
+        this.empList = []
+      }
+    )
+  }
+
+  selectEmpId(value: any) {
+    console.log("select value: ", value);
+
+    if (!this.selectedEmployee.includes(value.id)) {
+      this.selectedEmployee.push(value.id);
+    }
+
+    console.log("selectedEmp: ", this.selectedEmployee);
+  }
+
+  unselectEmpId(value: any) {
+    console.log("unselect value: ", value);
+    const index = this.selectedEmployee.indexOf(value.id);
+
+    if (index !== -1) {
+      this.selectedEmployee.splice(index, 1);
+    }
+
+    console.log("selectedEmp: ", this.selectedEmployee);
+  }
+
+  changeSiteLanguage(localeCode: string): void {
```

Comment:
[S] it was duplicate with search screen.
It should separate on another component